cmd/kubeadm/app/constants/constants.go

```diff
@@ -56,4 +56,18 @@ const (
 	// Minimum amount of nodes the Service subnet should allow.
 	// We need at least ten, because the DNS service is always at the tenth cluster clusterIP
 	MinimumAddressesInServiceSubnet = 10
+
+	// DefaultTokenDuration specifies the default amount of time that a bootstrap token will be valid
+	DefaultTokenDuration = time.Duration(8) * time.Hour
```

Comment:
Not sure what this comment means. Are you going to change this to 0?  I think that is probably a compatible default.